subprojects/core/src/integTest/groovy/org/gradle/api/tasks/DeleteTaskIntegrationTest.groovy

```diff
@@ -50,8 +50,8 @@ class DeleteTaskIntegrationTest extends AbstractIntegrationSpec {
 
     def "deleted files show up in task destroys"() {
         buildFile << """
-            import org.gradle.api.internal.tasks.properties.PropertyVisitor
-            import org.gradle.api.internal.tasks.properties.PropertyWalker
+            import org.gradle.internal.properties.PropertyVisitor
+            import org.gradle.internal.properties.bean.PropertyWalker
```

Comment:
Why is `PropertyWalker` in `bean` package but `PropertyVisitor` is not? Should that be in the same package?